deploy/tests/test_amis_snake.json

```diff
@@ -0,0 +1,19 @@
+[
```

Comment:
is there a script to produce this when the AMI changes? (or really, is there an easy way to get all this information given the AMI id/name)?

Reply:
I could/should create one.  I just did `aws ec2 describe-images` to get the AMI json and then munged it into the format that moto expects/accepts.  Before I create one, I'll ask the moto devs if they already have one.

Reply:
Don't even need to ask, just have to look.  oof. https://github.com/spulec/moto/blob/master/scripts/get_amis.py. 

I'll vendor that script into firesim/scripts with smarts to look at the AMI defined in awstools.py and add a python test that fails with a nicer message if the AMI file needs to be updated (otherwise, you'll just get an index error the first time we try to query it and index into the result that doesn't find the AMI we think should always be found).

Reply:
I'm almost done with creating this script but @davidbiancolin asked me to push it in a follow-on PR so that we can cut the release.  I'll open a follow-on PR later tonight that has it.

Reply:
Thanks Tim.